docs/content/mixins/kubernetes.md

```diff
@@ -7,13 +7,32 @@ description: Manage a set of Kubernetes manifests using the kubectl CLI
 
 Manage a set of Kubernetes manifests using the [kubectl CLI](https://kubernetes.io/docs/reference/kubectl/).
 
-Source: https://github.com/deislabs/porter/tree/main/pkg/kubernetes
+Source: https://github.com/deislabs/porter-kubernetes
```

Comment:
Can we also update the examples/usage section below? (Per https://github.com/deislabs/porter-kubernetes/issues/5)

Maybe the only needed update is to show how to set the clientVersion field on the mixin config?  Anything else?